app.js

```diff
@@ -0,0 +1,82 @@
+const bookCollection = document.querySelector('.bookshelf');
+const addButton = document.querySelector('.submit');
+const titleInput = document.getElementById('title');
+const authorInput = document.getElementById('author');
+
+const books = [];
+class BookCollection {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  getTitle() {
+    return this.title.value;
+  }
+
+  getAuthor() {
+    return this.author.value;
+  }
+}
+
+const newBook = new BookCollection(titleInput, authorInput);
+
+addButton.addEventListener('click', () => {
+  const titletext = newBook.getTitle();
+  const authorText = newBook.getAuthor();
+
+  if (titletext === '' || authorText === '') {
+    return;
+  }
+
+  books.push({ title: titletext, author: authorText });
+  let items = '';
+
+  books.forEach((item, index) => {
+    items += `<div class="coll${index} collection">
+    <div class='bookinfo'>
+    <p class='title'>"${item.title}" by</p>
+    <p class='author'>${item.author}</p>
+    </div>
+    <button class='remove' onclick="removeItem(${index})">remove</button>
+    </div>`;
+  });
+
+  bookCollection.innerHTML = items;
+  /* eslint-disable no-use-before-define */
+  saveData();
+  titleInput.value = '';
+  authorInput.value = '';
+});
+
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] [REQUIRED] Good job so far! Please fix linter errors without this kind of comment `/* eslint-disable ... */` because this type of comment is used in very few cases, and in your case, you can solve such errors yourself, and I think if implement my suggestion that this will increase your work experience and skills 😃